values/cert-manager/cert-manager.gotmpl

```diff
@@ -11,7 +13,7 @@ prometheus:
 
 podAnnotations:
   sidecar.istio.io/inject: "false"  
-
+{{- tpl $addIgnorePolicyAnnotation (list "psp-allowed-users" "psp-privileged" "psp-host-security") | nindent 2}}
```

Comment:
can you please indent this block to align with the annotation above? the contents can stay the same